apiserver/test/e2e/service_server_e2e_test.go

```diff
@@ -142,7 +143,7 @@ func TestCreateServiceV2(t *testing.T) {
 				require.NoError(t, err, "No error expected")
 				require.Nil(t, actualRPCStatus, "No RPC status expected")
 				require.NotNil(t, actualService, "A service is expected")
-				waitForRunningService(t, tCtx, actualService.Name)
+				waitForRayService(t, tCtx, actualService.Name, []rayv1api.ServiceStatus{rayv1api.NotRunning, rayv1api.Running})
```

Comment:
Can you use conditions instead? `ServiceStatus` will eventually be deprecated.

Reply:
Oh I just saw https://github.com/ray-project/kuberay/blob/8bb3222b3e36c099c7d10ae346aa3879a179288b/ray-operator/apis/ray/v1/rayservice_types.go#L16-L17, my bad